scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua

```diff
@@ -1,57 +1,145 @@
+      if(player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_ACCEPTED) then
+         player:startEvent(0x2798);
+      end
+   elseif(trade:hasItemQty(2955,10) == true and trade:getGil() == 0 and trade:getItemCount() == 10 and meritCount > 4) then
+      if(player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_ACCEPTED) then
+         player:startEvent(0x2799);
+     end
+   elseif(trade:hasItemQty(2955,1) == true and trade:hasItemQty(503,1) == true and trade:getGil() == 0 and trade:getItemCount() == 2 and meritCount > 9) then
+      if(player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_ACCEPTED) then
+         player:startEvent(10138);
+      end
+   end
 end;
-
------------------------------------
--- onTrigger Action
------------------------------------
-
+
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
 function onTrigger(player,npc)
+ 
+   if(player:hasKeyItem(LIMIT_BREAKER) == false and player:getMainLvl() >= 75) then
+      player:startEvent(0x273d);
+   elseif(player:getMainLvl() == 75 and player:levelCap() == 75 and MAX_LEVEL >= 80 and player:getQuestStatus(JEUNO,NEW_WORLDS_AWAIT) == QUEST_AVAILABLE) then
+      player:startEvent(0x273d,0,1,1,0);
+   elseif(player:getMainLvl() >= 76 and player:levelCap() == 80 and MAX_LEVEL >= 85 and player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_AVAILABLE) then
+      player:startEvent(0x273d,0,1,2,0);
+   elseif(player:getMainLvl() >= 81 and player:levelCap() == 85 and MAX_LEVEL >= 90 and player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_AVAILABLE) then
+      player:startEvent(0x273d,0,1,3,0);
+   elseif(player:getMainLvl() >= 91 and player:levelCap() == 90 and MAX_LEVEL >= 95 and player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_AVAILABLE) then
+      player:startEvent(0x273d,0,1,4,0);
+   elseif(player:getMainLvl() >= 91 and player:levelCap() == 95 and MAX_LEVEL >= 99 and player:getQuestStatus(JEUNO,BEYOND_INFINITY) == QUEST_AVAILABLE) then
+      player:startEvent(0x273d,0,1,5,0);
+   elseif(player:getQuestStatus(JEUNO,NEW_WORLDS_AWAIT) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,1,1);
+   elseif(player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,2,1);
+   elseif(player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,3,1);
+   elseif(player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,4,1);
+   elseif(player:getQuestStatus(JEUNO,BEYOND_INFINITY) == QUEST_ACCEPTED) then
+      player:startEvent(0x273d,0,1,5,1); -- player:startEvent(0x273d,0,1,6,1);
+   elseif(player:hasKeyItem(LIMIT_BREAKER) == true and player:getMainLvl() >= 75) then
+      player:startEvent(0x273d,0,1,0,0);
+   else
+      player:startEvent(0x273d,0,2,0,0);

```

Comment:
Your indents need to be 4 spaces per indent. Please. I'm begging. You have 3. 2 indents should = 8 spaces.